apps/src/lib/levelbuilder/lesson-editor/ActivityCard.jsx

```diff
@@ -0,0 +1,229 @@
+import React, {Component} from 'react';
```

Comment:
note to self: this file appears to have been copied from https://github.com/code-dot-org/code-dot-org/blob/615faba70d0876ed28d339e70721fa2bd075956d/apps/src/lib/levelbuilder/script-editor/LessonGroupCard.jsx

Reply:
Correct